code/modules/reagents/machinery/reagentgrinder.dm

```diff
@@ -44,13 +29,12 @@ var/global/list/juice_items = list (
 		/obj/item/stack/sheet/wax			  = list(WAX = 5),
 		/obj/item/candle					  = list(WAX = 1.25),
 		/obj/item/trash/candle				  = list(WAX = 1),
-		/obj/item/weapon/grown/nettle         = list(FORMIC_ACID = 0),
-		/obj/item/weapon/grown/deathnettle    = list(PHENOL = 0),
-		/obj/item/stack/sheet/charcoal        = list("charcoal" = 20),
+		/obj/item/stack/sheet/charcoal        = list(CHARCOAL = 20),
 		/obj/item/stack/sheet/bone	          = list(BONEMARROW = 20),
 
 		//Blender Stuff
-		/obj/item/weapon/reagent_containers/food/snacks/grown/soybeans = list(SOYMILK = -10), //I have no fucking idea what most of these numbers mean and I hate them.
+		//Inverted numbers add these reagents in proportion to the nutriment inside, and minus numbers multiply it, ..--==the more you know!*
```

Comment:
Inverted numbers vs minus numbers? Which adds, which multiplies? There are many confused people over this

Reply:
i meant to say zero for inverted, fixed